src/xo-app/host/tab-network.js

```diff
@@ -117,57 +172,105 @@ class PifItem extends Component {
     }
     return reconfigurePifIp(this.props.pif, { mode })
   }
-  _onEditIp = () => this._configIp('Static')
-
-  _editPif = vlan =>
-    editPif(this.props.pif, { vlan })
 
   render () {
-    const { networks, pif, vifsByNetwork } = this.props
+    const { pif } = this.props
     const { configModes } = this.state
+    return <Select
+      onChange={this._configIp}
+      options={configModes}
+      value={pif.mode}
+    >
+      {pif.mode}
+    </Select>
+  }
+}
+
+@connectStore(() => ({
+  vifsByNetwork: createGetObjectsOfType('VIF').groupBy('$network')
+}))
+class PifItemInUse extends Component {
```

Comment:
This is not directly related to the "in use" status of the PIF. It's a toggle component that's lets the user lock of unlock the network.